platform/jewel/ui/api-dump.txt

```diff
@@ -13,14 +24,35 @@ e:org.jetbrains.jewel.ui.component.AutoHideBehavior
 - s:getEntries():kotlin.enums.EnumEntries
 - s:valueOf(java.lang.String):org.jetbrains.jewel.ui.component.AutoHideBehavior
 - s:values():org.jetbrains.jewel.ui.component.AutoHideBehavior[]
+e:org.jetbrains.jewel.ui.component.ContextMenuItemActionType
+- java.lang.Enum
+- sf:COPY:org.jetbrains.jewel.ui.component.ContextMenuItemActionType
+- sf:CUT:org.jetbrains.jewel.ui.component.ContextMenuItemActionType
+- sf:PASTE:org.jetbrains.jewel.ui.component.ContextMenuItemActionType
+- sf:SELECT_ALL:org.jetbrains.jewel.ui.component.ContextMenuItemActionType
+- s:getEntries():kotlin.enums.EnumEntries
+- s:valueOf(java.lang.String):org.jetbrains.jewel.ui.component.ContextMenuItemActionType
+- s:values():org.jetbrains.jewel.ui.component.ContextMenuItemActionType[]
+f:org.jetbrains.jewel.ui.component.ContextMenuOption
+- androidx.compose.foundation.ContextMenuItem
+- sf:$stable:I
+- <init>(org.jetbrains.jewel.ui.icon.IconKey,org.jetbrains.jewel.ui.component.ContextMenuItemActionType,java.lang.String,kotlin.jvm.functions.Function0):V
+- b:<init>(org.jetbrains.jewel.ui.icon.IconKey,org.jetbrains.jewel.ui.component.ContextMenuItemActionType,java.lang.String,kotlin.jvm.functions.Function0,I,kotlin.jvm.internal.DefaultConstructorMarker):V
+- f:getActionType():org.jetbrains.jewel.ui.component.ContextMenuItemActionType
+- f:getIcon():org.jetbrains.jewel.ui.icon.IconKey
 org.jetbrains.jewel.ui.component.MenuScope
 - a:passiveItem(kotlin.jvm.functions.Function2):V
-- a:selectableItem(Z,org.jetbrains.jewel.ui.icon.IconKey,java.util.Set,kotlin.jvm.functions.Function0,Z,kotlin.jvm.functions.Function2):V
-- bs:selectableItem$default(org.jetbrains.jewel.ui.component.MenuScope,Z,org.jetbrains.jewel.ui.icon.IconKey,java.util.Set,kotlin.jvm.functions.Function0,Z,kotlin.jvm.functions.Function2,I,java.lang.Object):V
+- a:selectableItem(Z,org.jetbrains.jewel.ui.icon.IconKey,org.jetbrains.jewel.ui.component.ContextMenuItemActionType,java.util.Set,kotlin.jvm.functions.Function0,Z,kotlin.jvm.functions.Function2):V
```

Comment:
Can you make sure this is not a breaking change? E.g. by adding a new overload and deprecating the existing one